security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+Second, deanonymization is not binary.
+A user is not either fully anonymous or completely deanonymized.
+Instead, privacy research looks at anonymity compared to the _anonymity set_.
+
+The anonymity set is a central notion in privacy research.
+It refers to the set of identities such that, from an attacker's viewpoint, a given action could correspond to any of them.
+Consider a real-life example.
+Imagine you meet a person on a city street.
+What is their anonymity set from your point of view?
+If you don't know them personally, and without any additional information, their anonymity set roughly equals the city's population, including travelers.
+If you additionally consider their appearance, you may be able to roughly estimate their age and exclude the city residents who are obviously older or younger than the person in question from the anonymity set.
+Furthermore, if you notice that the person walks into the office of Company X using an electronic badge, the anonymity set shrinks to the number of Company X's employees and visitors.
+Finally, you may notice the license number of the car they used to arrive at the place.
+If you are a casual observer, this doesn't give you much.
+However, if you are a city official and have access to the database that matches license plate numbers with names, you can narrow down the anonymity set to just a few people: the car owner and their closest friends and relatives.
+
+This example illustrates a few important points.
+First, every bit of information may bring the adversary closer to their goal.
+It may not be necessary to shrink the anonymity set to the size of one.
+For instance, if the adversary plans a targeted DoS attack and can take down 100 servers, the anonymity set of 100 suffices.
+Second, the adversary can cross-correlate information from different sources.
+Even if a privacy leak looks relatively benign, we never know what it can achieve in combination with other data sources.
+Finally, especially in cryptographic settings, the attacker always has the "last resort" of brute force search.
+Cryptographic primitives are designed so that it is practically impossible to guess a secret such as a private key.
+Nevertheless, each bit of information brings the adversary closer to this goal, and at some point, it becomes attainable.
+
+In terms of Lightning, deanonymizing generally means deriving a correspondence between payments and users identified by node IDs.
+Each payment may be assigned a sender anonymity set and a receiver anonymity set.
+Ideally, the anonymity set consists of all the users of the network.
+This assures that the attacker has no information whatsoever.
+However, the real network leaks information that allows an attacker to narrow down the search.
+The smaller the anonymity set, the higher the chance of successful deanonymization.
+
+
+=== Differences between LN and Bitcoin in terms of privacy
+
+Bitcoin was initially perceived as an anonymous currency.
```

Comment:
Was it? Very strong statement to make here, Bitcoin has no explicit anonymity enhancing properties.

Reply:
ACK, even the comments under the first announcement on the cypherpunk mailing list were concerned about the weak privacy of this system.

Reply:
fixed. added reference